### PR TITLE
Video recorder/ziggeo init

### PR DESCRIPTION
**Pull request by mrJmek**
Документація: https://ssl.ziggeo.com/docs
Офіційний GitHub репозиторій: https://github.com/Ziggeo/ZiggeoRubySdk

- [ ] @vovka 
- [x] @AntonMayer 
- [ ] @superNazo

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,6 +11,8 @@ gem 'turbolinks',     '2.3.0'
 gem 'jbuilder',       '2.2.3'
 gem 'sdoc',           '0.4.0', group: :doc
```
Comment (vovka) [inline Gemfile]: Don't leave empty lines

File: app/views/video_recorder/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+</div><!-- container-fluid -->
+
+<div class="container-fluid">
+<div class="row">
+  <div class="col-xs-8 col-xs-offset-2">
+    <% @all_videos.each_slice(3) do |row_chunk| %>
+    <div class="row">
+      <% row_chunk.each do |video_token| %>
+      <div class="col-xs-4">
+        <%= raw('<ziggeo ziggeo-video="'+ video_token +'" ziggeo-width=320 ziggeo-height=240></ziggeo>') %>
```
Comment (vovka) [inline app/views/video_recorder/index.html.erb]: Why did you use `raw` here?

File: app/views/video_recorder/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      <% end %>
+    </div><!-- row -->
+  </br>
+  <% end %>
+</div><!-- col-xs-8 col-xs-offset-2 -->
+</div><!-- row -->
+</div><!-- container-fluid -->
+
+<div class="modal fade" id="videoRecorderModal" tabindex="-1" role="dialog" aria-labelledby="videoRecorderModalLabel">
+<div class="modal-dialog" role="document">
```
Comment (vovka) [inline app/views/video_recorder/index.html.erb]: What is this modal dialog for?

File: app/views/video_recorder/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    <div class="row">
+      <div class="col-xs-6 col-xs-offset-3">
+        <div class="wall-of-videos-container">
+          <ziggeo ziggeo-limit=15 ziggeo-width=320 ziggeo-height=240></ziggeo>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="modal-footer">
+    <button type="button" class="btn btn-primary" data-dismiss="modal" onClick="window.location.reload()">Close resorder</button>
```
Comment (vovka) [inline app/views/video_recorder/index.html.erb]: Don't use inline js. You'd ahve to move it to js file or remove this part with modal dialog at all.

File: app/views/video_recorder/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  </div>
+</div>
+</div>
+</div>
+
+<script>
```
Comment (vovka) [inline app/views/video_recorder/index.html.erb]: Please, move this to js file.

File: app/views/video_recorder/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+ZiggeoApi.Events.on("submitted", function (data) {
+  massage = '<div class="alert alert-success" role="alert">'
+    +'<button type="button" class="close" data-dismiss="alert" aria-label="Close"> <span aria-hidden="true">&times;</span> </button>'
```
Comment (vovka) [inline app/views/video_recorder/index.html.erb]: Don't use such html inside js string. It's not secured and is a bad isea in general.

**Comment by vovka** [comment]
Please add jira ticket number and title to this PR's title.

**Comment by vovka** [comment]
Please, use branch naming convention we were talking about: 
feature/description
bugfix/description

File: app/assets/stylesheets/video_recorder.scss [hunk 7]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the video-recorder controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (superNazo) [inline app/assets/stylesheets/video_recorder.scss]: do we need video_recorder.coffee and video_recorder.scss?

**Pull request closed without merging**